### PR TITLE
Make pending donation 3 percent to community correctly

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #2223

the pending donation for donate to Community (3 percent of donations) was creating incorrectly, so it was showing that donation in trace page but after mining transaction it was ok, but now we create pending transaction correctly

### CODE REVIEW

File: src/components/DonateButtonModal.jsx [hunk 1]
```diff
@@ -350,10 +350,20 @@ const DonateButtonModal = props => {
     adminId,
     _amount,
     donationOwnerAddress,
+    community,
```
Comment (aminlatifi) [inline src/components/DonateButtonModal.jsx:353]: Let's remove `community` and replace `adminId` with a structure holding more complete data of receiver project

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: src/components/DonateButtonModal.jsx (hunk 1 above)
Comment (mohammadranjbarz) [reply src/components/DonateButtonModal.jsx]: @aminlatifi 
It changed as you said

**Comment by aminlatifi** [comment]
Thanks @mohammadranjbarz

File: src/components/DonateButtonModal.jsx (hunk 1 above)
Comment (aminlatifi) [reply src/components/DonateButtonModal.jsx]: @mohammadranjbarz Thanks

**Pull request merged**